app/controllers/mixins/ems_common/angular.rb

```diff
@@ -69,7 +69,8 @@ def update_ems_button_save
       end
 
       def update_ems_button_validate
-        result, details = realtime_authentication_check
+        ems_type = find_record_with_rbac(model, params[:id])&.class
```

Comment:
There wasn't any handling of `nil` [here](https://github.com/ManageIQ/manageiq-ui-classic/pull/6667/files#diff-002200c01a8a53ff2a4439390e6bc3c4R77-R81) so I didn't go and check `nil` here assuming this will raise an exception if it can't find the record?  If so could I drop the safe navigator?

Reply:
Indeed, `app/controllers/mixins/checked_id_mixin.rb` - the method raises `ActiveRecord::RecordNotFound` if the record doesn't exist

Reply:
:+1: perfect thanks